apps/web-giddh/src/app/vouchers/preview/preview.component.ts

```diff
@@ -1,89 +1,865 @@
+    public selectedInvoice: any;
+    /** Hold invoice  type */
+    public voucherType: any = '';
+    /** Holds Total Results Count */
+    public totalResults: number = 0;
+    /** Holds params value */
+    public params: any = {};
+    /** Holds true show Payment Details enable */
+    public showPaymentDetails: boolean;
+    /** Holds true if company mode */
+    public isCompany: boolean;
+    /** Holds create new voucher text and url */
+    public createNewVoucher: any = {
+        text: '',
+        link: ''
+    };
+    /** Holds invoice type boolean status */
+    public invoiceType: any = {
+        isSalesInvoice: true,
+        isCashInvoice: false,
+        isCreditNote: false,
+        isDebitNote: false,
+        isPurchaseInvoice: false,
+        isProformaInvoice: false,
+        isEstimateInvoice: false,
+        isPurchaseOrder: false,
+        isReceiptInvoice: false,
+        isPaymentInvoice: false
+    };
+    /** Send Email Dialog Ref */
+    public sendEmailModalDialogRef: MatDialogRef<any>;
+
+    //============================
+    /** Holds page Size Options for pagination */
+    public pageSizeOptions: any[] = PAGE_SIZE_OPTIONS;
+    /**  */
+    public showEditMode: boolean = false;
+    /**  */
+    public isSendSmsEnabled: boolean = false;
+    /**  */
+    public isVoucherDownloading: boolean = false;
+    /**  */
+    public isVoucherDownloadError: boolean = false;
+    /**  */
+    public only4ProformaEstimates: boolean;
+    /**  */
+    public emailList: string = '';
+    /**  */
+    public moreLogsDisplayed: boolean = true;
+    /**  */
+    public voucherVersions: ProformaVersionItem[] = [];
+    /**  */
+    public filteredVoucherVersions: ProformaVersionItem[] = [];
+    /**  */
+    public invoiceDetailWrapperHeight: number;
+    /**  */
+    public invoiceDetailViewHeight: number;
+    /**  */
+    public invoiceImageSectionViewHeight: number;
+    /**  */
+    public isMobileView = false;
+    /**  */
+    public sessionKey$: Observable<string>;
+    /**  */
+    public companyName$: Observable<string>;
+    /**  */
+    public isFileUploading: boolean = false;
+    /** True, if attachment upload is to be displayed */
+    public shouldShowUploadAttachment: boolean = false;
+    /** Source of image to be previewed */
+    public imagePreviewSource: SafeUrl;
+    /** Stores the type of attached document for Purchase Record */
+    public attachedDocumentType: any;
+    /** Stores the BLOB of attached document */
+    private attachedDocumentBlob: Blob;
+    /** PDF file url created with blob */
+    public sanitizedPdfFileUrl: SafeUrl = null;
+    /** Attached PDF file url created with blob */
+    public attachedPdfFileUrl: any = '';
+    /** PDF src */
+    public pdfFileURL: string = '';
+    /** This will hold the attached file in Purchase Bill */
+    private attachedAttachmentBlob: Blob;
+    /** This will use for default template */
+    public thermalTemplate: any;
```

Comment:
```suggestion
    public defaultThermalTemplate: any;
```
Please change in overall file and also in html

Reply:
Done